src/main/java/org/opensearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -482,7 +484,7 @@ public String toString() {
 
     public static class OnBehalfOfSettings {
         @JsonProperty("enabled")
-        private Boolean oboEnabled = Boolean.TRUE;
+        private Boolean oboEnabled = Boolean.FALSE;
```

Comment:
Is this the behavior that we want with OBO? cc: @RyanL1997 since you worked on this feature